src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -65,4 +71,80 @@ public void testLoginAuthentication(){
         softAssertions.assertThat(responseElement.getPayload().isUid()).as("Email doesn't exists")
                 .isTrue();
     }
+
+    @Test
+    public void testNegativeLoginValidationWrongAppKey() {
+        String errorCode = "COMMONS-0001";
+        headerMap.put(map.get("gaValidationAppKeyHeaderName"), map.get("gaValidationAppKeyHeaderWrongValue"));
```

Comment:
instead of storing a random wrong value in json. Lets use the correct AppKey which is already stored in json, and then use `StringUtils` function from `org.apache.commons.lang3` to generate something random.